concepts/observability-custom-metrics/README.md

```diff
@@ -0,0 +1,93 @@
+# Custom workload metrics
+
+The following document presents possible alternatives for a Kyma user to store and analyze custom worload metrics. We want to address the following aspects:
+
+* How easy it is for a Kyma user to enable and adjust worload scraping?
+* Can we guarantee monitoring stack stability in case of a user mistake (e.g. which results in cardinality explosion). Monitoring and recoverability are important
+
+## Shared Prometheus
+
+Kyma comes with a pre-installed Prometheus. Currently, it is only used for monitoring Kyma components and it is not possible for it to scrape custom workload metrics. 
+The easiest way to let the customer monitor her worloads would be lifting the restrictions of Kyma Prometheus. However, in this case it's quite hard to guarantee its stability.
+
+One problem that can occur is so called cardinality explosion when the time series count grows because of high-cardinality labels. A good example is metrics exposed by Istio when elastic workloads scale up and down. Another very common use case is bad code deploys that stuff high-cardinality data (request IDs, timestamps, user-provided values, etc.) into one or more labels of one or more metrics.
+
+A cardinality explosion causes the following problems:
+
+* Increases Prometheus memory consumption, which can eventually lead to be OOM-killed
+* Increases scrape durations
+* Querying becomes effectively impossible
+
+Monitoring and preventing cardinality explosion is not an easy task. There is a project called [Bomb Squad](https://github.com/open-fresh/bomb-squad), which detects high-cardinality metrics and silences them by rewriting the scrpae config.
+However, it's an alpha project and is, in fact, abandoned. 
+
+### Pros
+
+* Easy to set up - just expose some Kyma Prometheus Operator features to end users
+
+### Cons
+
+* Since Prometheus Operator does not support annotation-based discovery of services, its CRD-based API (PodMonitor or ServiceMonitor) has to be exposed to end users. It is very flexible, but requires from end users advanced knowledge of Prometheus
```

Comment:
```suggestion
* Since Prometheus Operator does not support annotation-based discovery of services, its CRD-based API (Pod Monitor or Service Monitor) has to be exposed to end users. It is very flexible, but requires advanced knowledge of Prometheus.
```
I think Pod Monitor and Service Monitor are CRD. According to our [guides](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) should be written in natural language